test/dartdoc_test.dart

```diff
@@ -252,7 +252,7 @@ void main() {
       Package p = packageGraph.defaultPackage;
       expect(p.name, 'test_package');
       expect(p.hasDocumentationFile, isTrue);
-      expect(packageGraph.defaultPackage.publicLibraries, hasLength(10));
+      expect(packageGraph.defaultPackage.publicLibraries, hasLength(12));
```

Comment:
To the casual reader, this magic number looks scary.  Maybe a comment to reassure the unfamiliar?

Reply:
added